landing-page/content/common/how-to-release.md

```diff
@@ -222,6 +246,12 @@ Therefore, the release candidate is passed/rejected.
 
 After the release vote has passed, you need to release the last candidate's artifacts.
 
+But note that releasing the artifacts should happen around the same time the new docs are released
```

Comment:
This step can be just 

"Publish the new documentation and site docs which were prepared [here](/prepare-the-site-documentation-PRs). 

At line 178, we can have the section called.

`### prepare the site documentation PRs`

Note that releasing the artifacts should happen around the same time the new docs are released
so make sure the [documentation changes](/how-to-release/#documentation-release)
are prepared when going through the below steps and keep the PRs open for review.

Reply:
I wouldn't put the section to prepare the doc PRs before the voting is finished. You might end up preparing the PRs multiple times if more RCs are needed to pass the vote. I figured the ideal place to prepare the doc PRs is after the vote has passes and we started working on finishing the release.